crates/abq_cli/src/main.rs

```diff
@@ -363,6 +365,11 @@ async fn abq_main() -> anyhow::Result<ExitCode> {
                 Fixed(num) => num,
             };
 
+            let should_send_results = match rwx_access_token_kind.as_ref().map(String::as_ref) {
+                Some("personal_access_token") => false,
+                _ => true,
+            };
```

Comment:
We can have the type system do some of the verification work for us when deserializing an access token kind. For example: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=07d4dc13571804267d91abbb32f40db5

You could then change this to be

```suggestion
            let should_send_results = matches!(rwx_access_token_kind, AccessTokenKind::Personal);
```

Reply:
Awesome! Thanks :)